packages/scalars/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@powerhousedao/scalars",
+  "version": "0.0.0",
+  "license": "AGPL-3.0-only",
+  "private": false,
+  "files": [
+    "dist"
+  ],
+  "types": "dist/types/src/index.d.ts",
+  "main": "dist/cjs/src/index.js",
+  "module": "dist/es/src/index.js",
+  "type": "module",
+  "scripts": {
+    "check-types": "tsc",
+    "lint": "eslint . && npm run check-types",
+    "lint:fix": "eslint --fix . && npm run check-types",
+    "build": "NODE_ENV=production vite build",
+    "build:watch": "vite build --watch",
+    "test": "vitest run",
+    "clean": "rimraf dist",
+    "clean:node_modules": "rimraf node_modules"
+  },
+  "devDependencies": {
+    "@typescript-eslint/parser": "^6.19.0",
+    "@vitest/coverage-v8": "^2.0.5",
+    "glob": "^11.0.0",
+    "typescript": "^5.3.3",
+    "vite": "^5.0.11",
+    "vite-plugin-dts": "^3.7.1",
+    "vite-plugin-generate-file": "^0.1.1",
+    "vitest": "^2.0.5"
+  },
+  "dependencies": {
+    "graphql": "^16.9.0",
+    "zod": "^3.23.8"
+  },
```

Comment:
Graphql and zod should be peerDependencies instead, and excluded from the bundle.

They also need to be added as devDependencies to work locally.

Reply:
done!